swagger-java-discourse-client/api.json

```diff
@@ -0,0 +1,2212 @@
+{
```

Comment:
This should be in a resources directory

Reply:
I guess I followed a bad example :man_facepalming:

Reply:
Yeah, sorry. 
It's a useful convention though https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html